tooling/Accessibility.md

```diff
@@ -0,0 +1,7 @@
+# Accessibility Tooling
+
+## ANDI
+[ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) is the accessibility tool used by the NIH, and is used as our primary tool for discovering accessibility issues. To use, follow the installation instructions on the website linked, and then open the bookmark on the website you wish to analyze. Items you wish to analyze for accessibility issues will be highlighted.
```

Comment:
- NIT: It's technically a "Bookmarklet" (as the bookmark stores some javascript rather than a website)
- I think it might be worth calling out that there is a dropdown of different types of accessibility things to check (color, focusability, etc) and that you need to use that dropdown to get the appropriate type of accessibility you are trying to analyze

Reply:
Thank you for the feedback! I think the new commit addresses these concerns, but please let me know your thoughts!